### DISCUSSION TITLE
switching user (calendar)

### DISCUSSION THREAD

**Discussion by wobouser** [kimai/kimai#5262] [General]
### Describe the issue

hi,

i would like users to be able to see (readonly) the calendars of other users (timesheet entries and if possible absence/holiday).

in the documentation -> https://www.kimai.org/documentation/calendar.html

it says that I have to set the permission “view_other_timesheet” / timesheet (other) for the group.

now have the additional menu item “all times” for the users - which also provides no output (timesheet entries) when searching - regardless of whether it is a customer or a project.

so far i have only found information about kimai1 here.

### I already tried

- [X] I've read and searched [the documentation](https://www.kimai.org/documentation/).
- [X] I've searched for similar issues in this repository.
- [X] I've searched for similar issues in [the discussions](https://github.com/kimai/kimai/discussions).

### Kimai version

latest

### How do you run Kimai?

Other (please mention below)

### Which PHP version are you using?

8.1

**Comment by RoHaIDP** [comment]
Hi, we are also concerned about this point for our Kimai installation. We were trying out the absence/pto plugin in the online demo, and liked it a lot, however we require the possibility for users to see the absence/holiday entries of (all) other users.
